sstables/sstables.cc

```diff
@@ -1327,6 +1327,11 @@ future<> sstable::open_data() noexcept {
             _open = true;
             return make_ready_future<>();
         });
+    }).then([this] {
+        auto* ldcs = _components->scylla_metadata->data.get<scylla_metadata_type::LargeDataCounters, scylla_metadata::large_data_counters>();
+        if (ldcs) {
+            _large_data_counters.emplace(*ldcs);
+        }
```

Comment:
if we're opening a sstable just written, _large_data_counters already have the correct data. so we can just skip this part, right?

Reply:
disregard my comment above. _large_data_counters used during write is the one in sstable_writer, not the one in sstable class.

Reply:
Right. In the first version I initialized the one in the sstable and used, but I modified the flow to follow the existing design that collects the metadata in the writer and applies it to the sstable in `write_scylla_metadata`, to be reloaded by a following `open_data` that is called in any case.